packages/office-ui-fabric-react/src/components/ContextualMenu/ContextualMenu.types.ts

```diff
@@ -29,7 +28,7 @@ export interface IContextualMenu {
 
 }
 
-export interface IContextualMenuProps extends React.Props<ContextualMenu>, IWithResponsiveModeState {
+export interface IContextualMenuProps extends React.Props<any>, IWithResponsiveModeState {
```

Comment:
Definitely shouldn't be any. Make it an empty {} if it shouldn't correspond to anything.

Alternatively you could probably remove React.Props. It's really not supposed to extend that any more. It should certainly be removed in 6.0